pkg/kube/kube.go

```diff
@@ -149,6 +150,18 @@ func (k *KubeClient) GetApplication(name string) (*wego.Application, error) {
 	return &a, nil
 }
 
+func (k *KubeClient) LabelExistsInCluster(label string) error {
+	cmd := []string{"get", "app", "-l", fmt.Sprintf("weave-gitops.weave.works/app-identifier=%s", label)}
```

Comment:
Now I think we're only looking in the `default` namespace. I believe we should be using the namespace argument to `wego app add` which _defaults_ to `wego-system`